Graphics/Vulkan/VulkanGraphicsResourceMemoryRegion.cpp

```diff
@@ -1,56 +1,79 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#include <NovelRT/Exceptions/InitialisationFailureException.h>
+#include <NovelRT/Exceptions/InitialisationFailureException.hpp>
+#include <NovelRT/Graphics/GraphicsResource.hpp>
+#include <NovelRT/Graphics/GraphicsResourceMemoryRegion.hpp>
+
 #include <NovelRT/Graphics/Vulkan/VulkanGraphicsDevice.hpp>
 #include <NovelRT/Graphics/Vulkan/VulkanGraphicsResource.hpp>
 #include <NovelRT/Graphics/Vulkan/VulkanGraphicsResourceMemoryRegion.hpp>
 
-namespace NovelRT::Graphics::Vulkan
+namespace NovelRT::Graphics
```

Comment:
the namespace appears to be wrong? I could be wrong ofc here so figured I'd just ask.

Reply:
It's correct; GraphicsResourceMemoryRegion is in NovelRT::Graphics, this is just the vulkan specialization of it.